### DISCUSSION TITLE
Error adding any local tower loads to NRELOffshrBsline5MW_Onshore_ElastoDyn

### DISCUSSION THREAD

**Discussion by Veithh** [OpenFAST/openfast#2171] [Q&A]
Hey everyone, sorry if this might be a total beginners question. I am a student trying to teach myself OpenFast out of interest...
I am trying to add any local tower loads, gage 1-9 to my 5MW_Land_DLL_WTurb. Ive added the chan names from the OutListParameters.xlsx and after running the .fst its updated onto my Matlap ChanName list. Now if i want to plot it, it says "invalid" in the y-axis of the figure. Do I need to change settings in the NRELOffshrBsline5MW_Onshore_ElastoDyn or elsewhere? thank you, best regards from germany

**Comment by jjonkman** [comment]
Dear @Veithh,

When you use the local load outputs from ElastoDyn, you must identify the number of tower gage outputs you want (via ElastoDyn input `NTwGages`, up to 9) and select which nodes you want as output gages (via `TwrGagNd`, between 1 and `TwrNodes`).  If you use an output in the `OutList` for an output node that is higher than `NTwGages`, the output will be labeled as INVALID and the results will be zeroed.

Best regards,

**Reply by Veithh** [reply]
thank you worked out perfectly!
